contracts/Stage2/BondStorage.sol

```diff
@@ -30,6 +30,8 @@ contract BondStorage is AccessControl {
 
     modifier onlyWhitelisted() { require(hasRole(WHITELIST_BOND_STORAGE, msg.sender), "invalid-storage-operator"); _; }
 
+    modifier ifActive(address _user) { require(issuedBonds[_user].isActive); _; }
```

Comment:
since I "know" users will encounter this error, can you add a brief-and-descriptive revert message just as we use to do?

Reply:
oops! yes of course